htmresearch/frameworks/nlp/model_factory.py

```diff
@@ -35,6 +41,34 @@
 """
 
 
+
+class ClassificationModelTypes(object):
+  """ Enumeration of supported classification model types, mapping userland
+  identifier to constructor.  See createModel() for actual factory method
+  implementation.
+  """
+  CioWordFingerprint = ClassificationModelFingerprint
+  CioDocumentFingerprint = ClassificationModelFingerprint
+  Keywords = ClassificationModelKeywords
+  HTMNetwork = ClassificationModelHTM
+  DocumentFingerPrint = ClassificationModelDocumentFingerprint
+  cioword = ClassificationModelFingerprint
+  docfp = ClassificationModelDocumentFingerprint
+  htm = ClassificationModelHTM
+  keywords = ClassificationModelKeywords
+
+
+  @classmethod
+  def getTypes(cls):
+    types = {attr
```

Comment:
Seems like confusing logic for just returning a list of the model names. Can you make this simpler/easier to understand?